app/views/to_do_lists/create.js.erb

```diff
@@ -1,7 +1,7 @@
 var options = {
     <% if @list.persisted? %>
-        html: "<%=j render partial: 'list', locals: {list: @list} %>",
-        id: <%= @list.id %>,
+        html: "<%=j render partial: 'list', locals: {list: @list_presenter} %>",
+        id: <%= @list_presenter.id %>,
```

Comment:
@vlasiak I've outlined previously that it doesn't matter whether it is model or view object, it is just a list and that's it. I suggest to revert the name back to `list`.